.github/workflows/test.yaml

```diff
@@ -111,3 +134,9 @@ jobs:
           source venv/bin/activate
           pip install -e . -vv
           pytest -v .
+
+      - name: Run TPC-H examples
+        run: |
+          cd examples/tpch
```

Comment:
I *think* the first line here should activate the venv

```
      - name: Run TPC-H examples
        run: |\
          source venv/bin/activate
          cd examples/tpch
          python convert_data_to_parquet.py
          pytest _tests.py
```

Reply:
Thanks. I hadn't looked at how far it got before I had to step away for the day. I know I also need to add in the upload and download artifacts also